Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/SpinBox.cpp

```diff
@@ -1460,7 +1460,7 @@ QString DoubleSpinBox::stringValue(double value, bool truncated) const
         numDecimals = 0;
     }
 
-    return toString(value, numDecimals, locale(), isGroupSeparatorShown());
+    return toString(value, numDecimals, locale(), isGroupSeparatorShown(), false);
```

Comment:
question: This is likely intended it just seems odd on first glance that we don't pass the `truncated` parameter on line `1450` through to this call here. Are they different kind of truncation? Could the logic here be simplified if we just pass the bool through to `toString`?

Reply:
They are different. The value passed in means to either truncate to the number of decimals of to the numberofDisplayDecimals. I've reworked the logic so the term to pass is the round bool.

Reply:
Okay nice one, and thanks for the explanation 👍